docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,240 @@
+   with the version indicated on the `Tails website <https://tails.boum.org/index.en.html>`_.
+   If the installed Tail version is outdated, follow our :doc:`guide to updating Tails USBs <../update_tails_usbs>`.
+6. Run the command ``git status`` in the ``~/Persistent/securedrop`` directory.
+   The output should include the following text:
+
+   .. code-block:: none
+
+      HEAD detached at <version>
+
+   where ``<version>`` is  the version of the workstation code that is installed.
+   If the *Admin Workstation* is at |version|, it is up-to-date.
+7. If your SecureDrop code is outdated, follow our :doc:`upgrade guide <1.6.0_to_1.7.0>`
+   to perform a manual update. If that fails, please :ref:`contact us <contact_us>`
+   for assistance.
+8. (Recommended) Repeat this process for all *Admin Workstations* and *Journalist
+   Workstations*.
+
+.. note::
+
+   If your *Admin Workstation* is in an unrecoverable state, you can
+   follow our instructions to :doc:`rebuild an Admin Workstation <../rebuild_admin>`.
+
+.. _verify_ssh_access:
+
+Verify SSH access
+------------------
+Start up your *Admin Workstation* (with persistent storage unlocked) and run the
+following commands in a terminal:
+
+.. code:: sh
+
+  ssh app hostname     # command output should be 'app'
+  ssh mon hostname     # command output should be 'mon'
+
+If you are having trouble accessing the servers via SSH, try the following:
+
+- create a new Tor network circuit by disconnecting and reconnecting your
+  Internet link, and repeat the check
+- run the ``./securedrop-admin tailsconfig`` command and repeat the check
+- verify that the *Source* and *Journalist Interfaces* are available via their
+  desktop shortcuts
+- verify that the *Application* and *Monitor Servers* are up
+- :ref:`contact us <contact_us>` for assistance.
+
+.. _choose_migration_path:
+
+Choose Migration Path
+---------------------
+
+If your instance is already using v3 onion services, choose our
+:ref:`Standard Migration <migration_standard>` procedure.
+
+Instances that have not yet enabled v3 
+:ref:`onion services <glossary_onion_service>` and are nearing the
+April migration deadline have two options: 
+
+- :doc:`Upgrade to v3 onion services <../v3_services>` before performing 
+  the standard migration, or 
+- Take a backup of the current system, perform a fresh installation 
+  (which will be created with v3 onion services), and follow a 
+  modified restore path to restore only source and journalist 
+  data (:ref:`Alternate Migration <migration_alternate>` procedure).
+
+The second option is simpler, but potentially more disruptive to sources
+and journalists, since you will be abruptly switching from one set of
+onion URLs to another.
+
+
+Migration
+~~~~~~~~~
+
+.. _migration_standard:
+
+Standard Migration Procedure
+----------------------------
+(For SecureDrop instances already using v3 onion services)
+
+#. :doc:`Take a backup of the current instance <../backup_and_restore>`.
```

Comment:
If there are steps that are identical for both migration scenarios (e.g., make a backup, download/verify Ubuntu 20.04 media), it may make sense to split those into their own section for preparatory steps. I do recognize that means that earlier links won't be able to go directly to the path-specific section.

Reply:
For downloading/verifying Ubuntu 20.04, yes, I can add that to the preparatory steps. The reason I think that making a backup belongs during the actual migration window is it's time-sensitive; ideally the instance would be backed up and then taken offline immediately, so that no submissions fall through the cracks.

Reply:
Yes, that makes sense; I meant that the backup paragraph is duplicated in the two sections for the different paths, and if we can consolidate migration steps that are the same regardless of v2/v3 status, that might make it easier to maintain. But understand if you prefer to err on the side of just having each migration procedure be fully self-contained.